src/Modules/FudgeTools.psm1

```diff
@@ -795,6 +795,11 @@ function Add-PackagesFromNuspec
 # checks to see if the user has administrator privileges
 function Test-AdminUser
 {
+    if ($psVersionTable.Platform -eq 'Unix')
```

Comment:
Picky, but just to match the variable's name `$PSVersionTable`. Also, in case they ever change it, use `-ieq` to ignore case.